python/ql/test/experimental/library-tests/frameworks/flask/routing_test.py

```diff
@@ -0,0 +1,32 @@
+import flask
+
+from flask import Flask, make_response
+app = Flask(__name__)
+
+
+SOME_ROUTE = "/some/route"
+@app.route(SOME_ROUTE) # $routeSetup="/some/route"
+def some_route():  # $routeHandler
+    return make_response("some_route")
+
+
+# TODO: We should be able to handle this one
```

Comment:
Test expectation doesn't seem to indicate that we don't...?

Reply:
good catch :eyes: